app/src/main/java/sdp/moneyrun/game/GameBuilder.java

```diff
@@ -136,22 +140,21 @@ public GameBuilder setIsVisible(boolean isVisible) {
         return this;
     }
 
-    public void build() {
-
+    public Game build() {
         checkBuildArguments();
+        Game game;
 
-        if (riddles == null )
-            new Game(name, host, players, maxPlayerCount, startLocation, isVisible, coins);
-
-        else{
+        if (riddles == null ) {
+            game = new Game(name, host, players, maxPlayerCount, startLocation, isVisible, coins, DEFAULT_NUM_COINS, DEFAULT_RADIUS, DEFAULT_DURATION);
```

Comment:
Why is there no setter for the three default values? They should not be default values when the host creates the game

Reply:
I updated the gameBuilder properly